src/pages/frontpage/SearchSection.tsx

```diff
@@ -0,0 +1,92 @@
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { SearchTypeDropdown, SearchTypeValue } from '../search/SearchTypeDropdown';
+import { Box, Button, Typography } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { dataTestId } from '../../utils/dataTestIds';
+import { SearchTextField } from '../search/SearchTextField';
+import EastIcon from '@mui/icons-material/East';
+import { UrlPathTemplate } from '../../utils/urlPaths';
+
+const getCorrectParameters = (inputValue: string, searchType: SearchTypeValue) => {
+  switch (searchType) {
+    case SearchTypeValue.Result:
+      return inputValue ? `query=${inputValue}` : '';
+    case SearchTypeValue.Person:
+      return `type=person${inputValue ? `&name=${inputValue}` : ''}`;
+    case SearchTypeValue.Project:
+      return `type=project${inputValue ? `&multiple=${inputValue}` : ''}`;
+    default:
+      return '';
+  }
+};
```

Comment:
Vi har veldig mye av disse rundt omkring, men på nye ting prøver vi å gå bort fra manuell strengmanipulering til å heller bruke objektet som er laget for dette. Da slipper vi å passe på syntaksen med "?", "&" og sånt, i tillegg til at det er enklere å gjenbruke nøkkel uten å måtte ekstra triksing

```suggestion
const getSearchParams = (inputValue: string, searchType: SearchTypeValue) => {
  const searchParams = new URLSearchParams();
  switch (searchType) {
    case SearchTypeValue.Result:
      if (inputValue) searchParams.append(ResultParam.Query, inputValue);
      break;
    case SearchTypeValue.Person:
      if (inputValue) searchParams.append(PersonSearchParameter.Name, inputValue);
      break;
    case SearchTypeValue.Project:
      if (inputValue) searchParams.append(ProjectSearchParameter.Multiple, inputValue);
      break;
  }
  return searchParams;
```